app/interactors/projects/destroy.rb

```diff
@@ -0,0 +1,13 @@
+module Projects
+  class Destroy
+    include Interactor
+
+    delegate :project, to: :context
+
+    def call
+      context.project = project
+
+      context.fail!(error: "Project destruction failed") unless project.destroy
```

Comment:
Redundant failed context, because you anyway can destroy project

```suggestion
      project.destroy
```